sscanss/app/commands/insert.py

```diff
@@ -133,33 +133,34 @@ class InsertTomographyFromFile(QtWidgets.QUndoCommand):
     :param sizes_and_centres: array of pixel positions (if loading a stack of TIFF's) or empty (if loading a nexus file)
     :type sizes_and_centres: List[float, float, float, float, float, float]
     """
-    def __init__(self, filepath, presenter, sizes_and_centres=None):
+    def __init__(self, filepath, presenter, pixel_sizes=None, pixel_centres=None):
         super().__init__()
 
         self.filepath = filepath
         self.presenter = presenter
-        self.sizes_and_centres = sizes_and_centres
+        self.pixel_sizes = pixel_sizes
+        self.pixel_centres = pixel_centres
         self.old_volume = None
 
     def redo(self):
         """Using a worker thread to load in tomography data"""
         self.presenter.view.progress_dialog.showMessage('Loading Tomography Data')
         self.old_volume = self.presenter.model.volume
-        self.worker = Worker(self.loadTomo, [self.sizes_and_centres])
+        self.worker = Worker(self.loadTomo, [self.pixel_sizes])
         self.worker.job_succeeded.connect(self.onImportSuccess)
         self.worker.finished.connect(self.presenter.view.progress_dialog.close)
         self.worker.job_failed.connect(self.onImportFailed)
         self.worker.start()
 
-    def loadTomo(self, sizes_and_centres=None):
+    def loadTomo(self, pixel_sizes=None):
```

Comment:
Sorry I missed this the argument is not required. please remove argument and docstring